hyp3_testing/util.py

```diff
@@ -0,0 +1,29 @@
+import json
+import random
+import string
+
+from jinja2 import Environment, PackageLoader, StrictUndefined, select_autoescape
+
+
+def get_job_name() -> str:
```

Comment:
`generate` instead of `get`? I initial expected this function to decode something